### PR TITLE
[sonic-py-common] Added an API to get file path containing SONiC version

### PR DESCRIPTION
**Pull request by smaheshm**
**- Why I did it**

Added an API to get the file path containing SONiC version so that the API can be mocked during unit tests.

**- How I did it**

Added the API.

**- How to verify it**

Manual verification.

```
Last login: Mon Dec 28 18:32:34 2020 from 100.127.20.23
admin@str-s6000-acs-9:~$ python3
Python 3.7.3 (default, Jul 25 2020, 13:03:44) 
[GCC 8.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from sonic_py_common import device_info                                                                                                                                           
>>> device_info.get_sonic_version_file()
'/etc/sonic/sonic_version.yml'
>>> 
```

**- Which release branch to backport (provide reason below if selected)**

- [ ] 201811
- [ ] 201911
- [ ] 202006

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Comment by jleveque** [comment]
The new `get_sonic_version_file()` function isn't called anywhere. How do we benefit from mocking it in unit tests?

**Comment by smaheshm** [comment]
> The new `get_sonic_version_file()` function isn't called anywhere. How do we benefit from mocking it in unit tests?

It will be used in 'sonic-utilities/config'.

https://github.com/Azure/sonic-utilities/pull/1266 

checkout config/main.py.

**Comment by smaheshm** [comment]
retest mellanox please

**Comment by smaheshm** [comment]
retest vsimage please

**Comment by smaheshm** [comment]
retest mellanox please

File: src/sonic-py-common/sonic_py_common/device_info.py [hunk 1]
```diff
@@ -297,6 +297,11 @@ def get_sonic_version_info():
 
     return data
 
+def get_sonic_version_file():
+    if not os.path.isfile(SONIC_VERSION_YAML_PATH):
+        return None
```
Comment (judyjoseph) [inline src/sonic-py-common/sonic_py_common/device_info.py:302]: Does all the current callers of this API, handle a return value of "None"  ? If this is a mandatory file, we could raise an exception.

Comment (smaheshm) [reply src/sonic-py-common/sonic_py_common/device_info.py:302]: There is only one caller (https://github.com/Azure/sonic-utilities/pull/1266). I'd say let's leave it to the caller to decide. Python will raise exception anyway.

**Comment by smaheshm** [comment]
retest mellanox please

**Review by judyjoseph — APPROVED** [review]

**Pull request merged**